.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@ Please cherry-pick my commits into:
 
 * [ ] Foreman 3.5/Katello 4.7
```

Comment:
Thanks for the PR! Should we add this as well?
```suggestion
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
```

Reply:
there is no such thing as Satellite 6.13 yet, so I refrained from doing it.

we could say "planned Satellite 6.13"?

Reply:
Sure.